### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by frdmn**
Feel free to reject in case you do not want this added in the repository.

### CODE REVIEW

**Comment by joshghent** [comment]
I got the following errors from my Pi (had to change the image to arm32v7/golang:1.13)

Any ideas?

```
Sending build context to Docker daemon  165.9kB
Step 1/6 : FROM arm32v7/golang:1.13
 ---> 19643ce633cc
Step 2/6 : RUN mkdir /app
 ---> Using cache
 ---> c8cecad304d4
Step 3/6 : ADD . /app/
 ---> Using cache
 ---> 46e05afa3757
Step 4/6 : WORKDIR /app
 ---> Using cache
 ---> 58db8088c4a0
Step 5/6 : RUN go build -o gramarr .
 ---> Running in ccb7a657596a
config.go:9:2: cannot find package "github.com/alcmoraes/gramarr/radarr" in any of:
	/usr/local/go/src/github.com/alcmoraes/gramarr/radarr (from $GOROOT)
	/go/src/github.com/alcmoraes/gramarr/radarr (from $GOPATH)
config.go:10:2: cannot find package "github.com/alcmoraes/gramarr/sonarr" in any of:
	/usr/local/go/src/github.com/alcmoraes/gramarr/sonarr (from $GOROOT)
	/go/src/github.com/alcmoraes/gramarr/sonarr (from $GOPATH)
conversation.go:7:2: cannot find package "github.com/patrickmn/go-cache" in any of:
	/usr/local/go/src/github.com/patrickmn/go-cache (from $GOROOT)
	/go/src/github.com/patrickmn/go-cache (from $GOPATH)
conversation.go:6:2: cannot find package "gopkg.in/tucnak/telebot.v2" in any of:
	/usr/local/go/src/gopkg.in/tucnak/telebot.v2 (from $GOROOT)
	/go/src/gopkg.in/tucnak/telebot.v2 (from $GOPATH)
```

**Comment by adampetrovic** [comment]
@joshghent The Dockerfile needs to run `go get` before `go build` will work.

```
$ go build -o gramarr .
config.go:9:2: cannot find package "github.com/alcmoraes/gramarr/radarr" in any of:
	/usr/local/Cellar/go/1.13.4/libexec/src/github.com/alcmoraes/gramarr/radarr (from $GOROOT)
	/Users/apetrovic/code/go/src/github.com/alcmoraes/gramarr/radarr (from $GOPATH)
config.go:10:2: cannot find package "github.com/alcmoraes/gramarr/sonarr" in any of:
	/usr/local/Cellar/go/1.13.4/libexec/src/github.com/alcmoraes/gramarr/sonarr (from $GOROOT)
	/Users/apetrovic/code/go/src/github.com/alcmoraes/gramarr/sonarr (from $GOPATH)
conversation.go:7:2: cannot find package "github.com/patrickmn/go-cache" in any of:
	/usr/local/Cellar/go/1.13.4/libexec/src/github.com/patrickmn/go-cache (from $GOROOT)
	/Users/apetrovic/code/go/src/github.com/patrickmn/go-cache (from $GOPATH)
conversation.go:6:2: cannot find package "gopkg.in/tucnak/telebot.v2" in any of:
	/usr/local/Cellar/go/1.13.4/libexec/src/gopkg.in/tucnak/telebot.v2 (from $GOROOT)
	/Users/apetrovic/code/go/src/gopkg.in/tucnak/telebot.v2 (from $GOPATH)
$ go get
$ go build -o gramarr .
```

Alternatively, if the project switches to using modules, then the build command will work.

I have sent up a PR to start using modules: https://github.com/alcmoraes/gramarr/pull/6

**Comment by frdmn** [comment]
@joshghent Can you retry?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM golang:onbuild
+RUN mkdir /app
+ADD . /app/
+WORKDIR /app
+RUN go get
```
Comment (ChaosMarc) [inline Dockerfile:5]: I had to add the following line to be able to build the docker image:
`ENV GOBIN=$GOPATH/bin`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM golang:onbuild
```
Comment (ChaosMarc) [inline Dockerfile:1]: `FROM golang:onbuild`
is deprecated and should be replaced by 
`FROM golang`
or 
`FROM golang:alpine`

**Review by ChaosMarc — CHANGES_REQUESTED** [review]
With minor adjustments I was able to test the dockerfile and could run in successfully (Win10 x64 + armhf v7).

**Comment by anarqz** [comment]
Thanks guys! Made a few changes in README as well, tested and its ready to go!

**Pull request merged**